content/intro-to-storybook/vue/es/simple-component.md

```diff
@@ -0,0 +1,254 @@
+---
+title: 'Construye un componente simple'
```

Comment:
forgot about this...sorry...pluralize the title to "Construye un componente simples" and the toc title also needs to be pluralized should be "tocTitle: 'Componente Simples'